src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -164,9 +164,8 @@ private static function parseDsn($dsn)
     {
         $engine = substr($dsn, 0, strpos($dsn, ':'));
         $tags = ['db.engine' => $engine];
-        $tags[Tag::DB_SYSTEM] = isset(self::DB_DRIVER_TO_SYSTEM[$engine])
-            ? self::DB_DRIVER_TO_SYSTEM[$engine]
-            : 'other_sql';
+        $db_system = self::DB_DRIVER_TO_SYSTEM[$engine];
+        $tags[Tag::DB_SYSTEM] = isset($db_system) ? $db_system : 'other_sql';
```

Comment:
This will generate a warning, which we swallow, but does this really fix an issue with the compiler? Seems odd, the former looks correct.

Reply:
I've now replaced this with just `$tags[Tag::DB_SYSTEM] = self::DB_DRIVER_TO_SYSTEM[$engine] ?? 'other_sql` but still getting the build errors. This was the only change that was made, the currently failing tests were passing yesterday when this line was different :thinking_face:

Reply:
Yes, we've been rebuilding the docker containers, needs #1922 to be merged.

Reply:
If you rebase it on master, then build should be green now.